src/pages/employee/employees/index.tsx

```diff
@@ -0,0 +1,48 @@
+import EmployeeHeader from "@/components/EmployeeHeader";
+import MyHead from "@/components/MyHead";
+import EmployeeShow from "@/components/UserShow";
```

Comment:
名前を変えてインポートすると後で混乱しやすいので、なるべく同じ名前でインポートすることをおすすめします。

```suggestion
import UserShow from "@/components/UserShow";
```

Reply:
https://github.com/tatata05/next-app/pull/11/commits/4e78a323539077b82fbd813014429d0889ff60a1
上記と同じコミットですが、修正しました！